plugins/samanage/help.md

```diff
@@ -14,18 +14,31 @@ This plugin utilizes the [Samanage API](https://www.samanage.com/api).
 
 # Requirements
 
-* A Samanage token
+* A Solarwinds Service Desk API token
+
+# Supported Product Versions
+
+* Solarwinds Service Desk with API version 2.1
 
 # Documentation
 
 ## Setup
 
 The connection configuration accepts the following parameters:
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|token|credential_secret_key|None|True|API Token generated by Samanage admin|None|
-|eu_customer|boolean|False|True|Is the customer based in Europe?|None|
+|Name|Type|Default|Required| Description                             |Enum|Example|
+|----|----|-------|--------|-----------------------------------------|----|-------|
+|eu_customer|boolean|False|True| Is the customer based in Europe?        |None|False|
```

Comment:
It seems PyCharm reformatted those lines. Could you please remove that spaces?

Reply:
Fixed.